src/IconToggle/IconToggle.react.js

```diff
@@ -33,7 +33,7 @@ const propTypes = {
     /**
     * Name of icon to show
     */
-    name: PropTypes.string.isRequired,
+    name: PropTypes.string,
```

Comment:
Hmmm, I think @chirag04 is right. It seems like a bug in `ListItem` rather. It doesn't make any sense to use `IconToggle` without `name`.

We probably need to use:
```js
                <IconToggle
                    key={action}
                    color={flattenRightElement.color}
                    onPress={() => this.onRightElementPressed({ action })}
                    name={action}
                    style={styles.rightElement} 
                />
```

instead of this:
```js
                <IconToggle
                    key={action}
                    color={flattenRightElement.color}
                    onPress={() => this.onRightElementPressed({ action })}
                >
                    <Icon name={action} size={24} style={styles.rightElement} />
                </IconToggle>
```

What do you think guys?